main.go

```diff
@@ -257,12 +258,14 @@ func main() { // nolint:gocyclo
 		setupLog.Error(err, "unable to create controller", "controller", "ToolchainConfig")
 		os.Exit(1)
 	}
+
 	if err := (&toolchainstatus.Reconciler{
-		Client:         mgr.GetClient(),
-		Scheme:         mgr.GetScheme(),
-		HTTPClientImpl: &http.Client{},
-		GetMembersFunc: commoncluster.GetMemberClusters,
-		Namespace:      namespace,
+		Client:              mgr.GetClient(),
+		Scheme:              mgr.GetScheme(),
+		HTTPClientImpl:      &http.Client{},
+		GetGithubClientFunc: commonclient.NewGitHubClient,
```

Comment:
Yes, I could do this, but I've realized that it would be easier to just pass the `AccessTokenKey` value to this function, since I already have the `toolchainConfig` at that point : https://github.com/codeready-toolchain/host-operator/pull/819/files#diff-dbe76ee9dfabe95c78e8d2e7c3efdf6526bfd4719168b8529c518129cb83a6acR322-R346

Maybe I could refactor the `isProdEnvironment` environment function, but doing so it avoids checking twice for the same potential error returned by `GetToolchainConfig`.